input/resources/au-core-requester.xml

```diff
@@ -216,7 +216,7 @@
 <td class="text-center">y</td>
 <td class="text-center"></td>
 <td class="text-center"></td>
-<td>class, date, discharge-disposition, identifier, location, patient, patient.identifier, patient+class, patient+date, patient+discharge-disposition, patient+location, patient+status, patient+type, status, type</td>
+<td>class, date, identifier, location, patient, patient.identifier, patient+class, patient+date, patient+location, patient+status, status</td>
 <td/>
 <td>
```

Comment:
AU Core Requester - need to remove AU Core MedicationRequest search parameters category and identifier from the narrative content.

(Moved the comment raised by @dbojicic here so that the conversation can be tracked and resolved)

Reply:
fixed.